<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>genericOnlineSearch</class>
 <widget class="QWidget" name="genericOnlineSearch">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Online drug information</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>drugs.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rxlist.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fda.gov</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>usnlm.gov</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wikipedia.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>google.com</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="buttonSearch">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QListView" name="listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="clickableLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>clickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
